bindings_test.go

```diff
@@ -1120,18 +1127,27 @@ func TestLOBRetrievalWithJSON(t *testing.T) {
 }
 
 func testLOBRetrieval(t *testing.T, useArrowFormat bool) {
-	// the LOB sizes to be tested
-	testSizes := [5]int{smallSize, originSize, mediumSize, largeSize, maxLOBSize}
-	var res string
-
 	runDBTest(t, func(dbt *DBTest) {
+		parameters := dbt.connParams()
+		varcharBinaryMaxSizeRaw := parameters[maxVarcharAndBinarySizeParam]
+		if varcharBinaryMaxSizeRaw != nil && *varcharBinaryMaxSizeRaw != "" {
+			varcharBinaryMaxSize, err := strconv.Atoi(*varcharBinaryMaxSizeRaw)
+			if err == nil {
```

Comment:
Why do we need this ifs in which case it can happen? Some specific env?

Reply:
For now we have no guarantee that this parameter will be returned from server via /login response, since it depends on FeatureWithClientEnabled matrix.

Reply:
Ok, but the value is either not returned (two lines above) or it is returned and has proper format, right? It should never be returned but in a form in which you can't convert it to int, right?

Reply:
This is already changed.